adminSiteServer/apiRouter.ts

```diff
@@ -2008,30 +2011,47 @@ postRouteWithRWTransaction(
     apiRouter,
     "/tags/new",
     async (req: Request, res, trx) => {
-        const tag = (req.body as { tag: any }).tag
+        const tag = req.body
+        function validateTag(
+            tag: unknown
+        ): tag is { name: string; slug: string | null } {
+            return (
+                checkIsPlainObjectWithGuard(tag) &&
+                typeof tag.name === "string" &&
+                (tag.slug === null || typeof tag.slug === "string")
```

Comment:
Here it probably makese sense to also check if the slug is not empty string, just in case